openslides_backend/migrations/migrations/0034_rename_change_default_projector_ids.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Dict, List, Optional, cast
+
+from datastore.migrations import (
+    BaseEvent,
+    BaseMigration,
+    CreateEvent,
+    DeleteFieldsEvent,
+    ListUpdateEvent,
+    UpdateEvent,
+)
+from datastore.shared.util import collection_from_fqid
+
+from openslides_backend.models.models import Meeting
+
+
+class Migration(BaseMigration):
+    """
+    This migration renames default_projector_$.._id into
+      default_projector_$.._ids and changes value to list.
+    """
+
+    target_migration_index = 35
+    collection = "meeting"
+
+    def modify(self, object: Dict[str, Any]) -> None:
+        if "default_projector_$_id" in object:
+            object["default_projector_$_ids"] = object["default_projector_$_id"]
+            del object["default_projector_$_id"]
+        for name in cast(List[str], Meeting.default_projector__ids.replacement_enum):
+            old_field = f"default_projector_${name}_id"
+            new_field = f"default_projector_${name}_ids"
+            if old_field in object:
```

Comment:
Iterate over the object's fields instead and check if they start with `default_projector_$`. This way, you can remove the dependency to the `Meeting` object.

Reply:
Done.